list2/ex25/index.js

```diff
@@ -0,0 +1,26 @@
+import { generateMatrix, printMatrix } from "../helper.js";
+
+export function columnSumOfMatrix() {
+  const rowCount = 15;
+  const columnCount = 20;
+  const columnCountArray = [];
```

Comment:
This will create an array with 20 zero elements.

```suggestion
  const columnCountArray = Array.from({ length: columnCount }, () => 0)
```

Reply:
this will help me a lot in other exercises, thank you!